### PR TITLE
Reader: fix position of visit link icon

### PR DESCRIPTION
**Pull request by bluefuton**
We have a regression on Reader posts cards (both combined cards and regular post cards) meaning that the visit link icon displays in the wrong position (see https://github.com/Automattic/wp-calypso/issues/18143).

Before:

30658904-17208b42-9df1-11e7-9db5-952fc0253c6a

After:

screen shot 2017-09-22 at 18 47 53
screen shot 2017-09-22 at 18 47 57

### CODE REVIEW

**Comment by fraying** [comment]
Thanks for working on this, @bluefuton. Please note that it's also happening on the normal cards: https://github.com/Automattic/wp-calypso/issues/18143#issuecomment-330989563

File: client/blocks/reader-combined-card/style.scss [hunk 1]
```diff
@@ -180,17 +181,17 @@
...
 
-	&:not( .is-placeholder )::after {
+	&:not(.is-placeholder)::after {
 		@include long-content-fade( $size: 20% );
 	}
 }
 
 .reader-combined-card .reader-visit-link {
-	margin-left: -9px; // override gridicon margin
+	margin-left: 0;
```
Comment (bluefuton) [inline client/blocks/reader-combined-card/style.scss]: Fix gridicon margin

File: client/blocks/reader-combined-card/style.scss [hunk 2]
```diff
@@ -180,17 +181,17 @@
...
 	margin-right: 26px;
 
 	.reader-visit-link__label {
-		margin-left: 4px;
+		margin-left: -4px;
```
Comment (bluefuton) [inline client/blocks/reader-combined-card/style.scss]: Fixes label margin

File: client/blocks/reader-post-card/style.scss [hunk 3]
```diff
@@ -556,16 +548,15 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	height: 22px;
 
 	&.reader-post-actions__visit {
-
 		.gridicon {
 			position: relative;
-				left: -8px;
-				top: -1px;
+			left: -2px;
```
Comment (bluefuton) [inline client/blocks/reader-post-card/style.scss]: Fixes gridicon margin

File: client/blocks/reader-post-card/style.scss [hunk 4]
```diff
@@ -556,16 +548,15 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
...
+			top: -1px;
 		}
 	}
 
 	.gridicons-external {
```
Comment (bluefuton) [inline client/blocks/reader-post-card/style.scss]: Regular cards: fixes label margin

File: client/blocks/reader-post-card/style.scss [hunk 5]
```diff
@@ -706,17 +693,17 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
...
 
-	@media #{$reader-post-card-breakpoint-small}  {
+	@media #{$reader-post-card-breakpoint-small} {
 		margin: 0 0 10px;
 	}
 }
 
 .reader-post-card__gallery-item {
-	border: 1px solid lighten( $gray, 30% );
+	border: 1px solid lighten($gray, 30%);
```
Comment (samouri) [inline client/blocks/reader-post-card/style.scss]: seems like we lost a lot of whitespace in this file around parens

File: client/blocks/reader-post-card/style.scss [hunk 6]
```diff
@@ -257,7 +249,7 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	margin-right: 20px;
 	cursor: pointer;
 	position: relative;
-		top: 0;
+	top: 0;
```
Comment (samouri) [inline client/blocks/reader-post-card/style.scss]: :O

**Comment by samouri** [comment]
Made some whitespace update and removed the `@format` tag which we shouldn't be applying to css.

Checking that the fix still works and then I'll 🚢

**Comment by bluefuton** [comment]
Thanks for getting this over the finish line @samouri 👍 

Is it possible for us to turn off CSS formatting in https://github.com/Automattic/calypso-prettier, or should we be handling this in our own editors?

**Comment by samouri** [comment]
@bluefuton: right now I believe its something we should be handling in our own editors.
If there are any editors in which its difficult to configure this way let me know and I'll look into adding the support into calypso-prettier.

Personally I just don't have `formatOnSave` turned on.

**Comment by bluefuton** [comment]
Cool, thanks @samouri. I now have this set up in Sublime:
```
{
	"auto_format_on_save": true,
	"auto_format_on_save_excludes": ["*.scss", "*.json"],
}
```

**Comment by jancavan** [comment]
@bluefuton 

> "auto_format_on_save": true,
	"auto_format_on_save_excludes": ["*.scss", "*.json"],

I added this to Sublime preferences, but it's not formatting my code the way it's done them in this PR? Anything I'm missing?

**Comment by bluefuton** [comment]
> I added this to Sublime preferences, but it's not formatting my code the way it's done them in this PR? Anything I'm missing?

Hey @jancavan - those preferences should exclude .scss files, so they _won't_ get auto-formatted :) The CSS formatting behaviour was a bit quirky, so I wanted to turn it off. If you have those lines in your prefs, .jsx files should be auto-formatted, though.

**Pull request merged**